ui/app/(wrapped)/pilotage-reforme/types.ts

```diff
@@ -0,0 +1,29 @@
+import { api } from "../../../api.client";
+
+export type PilotageReformeStatsQuery = Parameters<
+  typeof api.getPilotageReformeStats
+>[0]["query"];
+
+export type PilotageReformeStatsRegionsQuery = Parameters<
+  typeof api.getPilotageReformeStatsRegions
+>[0]["query"];
+
+export type Filters = Pick<
+  PilotageReformeStatsQuery,
+  "codeNiveauDiplome" | "codeRegion"
+>;
+
+export type FiltersRegions = Pick<
+  PilotageReformeStatsRegionsQuery,
+  "codeNiveauDiplome"
+>;
+
+export type Order = Pick<PilotageReformeStatsRegionsQuery, "order" | "orderBy">;
+
+export type PilotageReformeStats = Awaited<
+  ReturnType<ReturnType<typeof api.getPilotageReformeStats>["call"]>
+>;
```

Comment:
Et pareil en dessous 

```suggestion
export type PilotageReformeStats = ApiType<typeof api.getPilotageReformeStats>;
```